ReleaseProcessAutomation/Jira/JiraFunctionality.cs

```diff
@@ -55,6 +61,39 @@ public void CreateAndReleaseJiraVersion (SemanticVersion currentVersion, Semanti
     ReleaseVersion(currentVersionID, nextVersionID, squashUnreleased);
   }
 
+  public bool MoveClosedIssuesToNewVersion (SemanticVersion currentVersion, SemanticVersion nextVersion)
+  {
+    var currentJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(currentVersion.ToString());
+    var nextJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(nextVersion.ToString());
+    if(currentJiraVersion == null || nextJiraVersion == null)
+    {
+      _console.WriteLine($"Could not find current jira version '{currentVersion}' or next jira version '{nextVersion}'");
+      return false;
+    }
+    
+    var closedIssues = _jiraIssueService.FindAllClosedIssues(currentJiraVersion.id);
+    _jiraIssueService.MoveIssuesToVersion(closedIssues, currentJiraVersion.id, nextJiraVersion.id);
+    return true;
+  }
+
+  public IEnumerable<JiraToBeMovedIssue> GetClosedVersionsSummaryWithExactFixVersion (SemanticVersion version)
```

Comment:
As it is, i don't understand what the method does just from the name. Maybe something like `GetIssuesForFixVersion` or something would be better.